### DISCUSSION TITLE
Question about kappa's value in input file

### DISCUSSION THREAD

**Discussion by abc-hy** [idaholab/moose#20052] [Q&A Modules: Phase field] [answered]
Dear Sir/Madam,

Could you please explain the kappa's meaning to me in the input file? What does kappa represent and how does it relate to the interface width and energy? Normally, how to set the value of kappa?

![image](https://user-images.githubusercontent.com/63679626/149678900-7a001f33-b9b8-4b8f-b94c-a03ca912e7a7.png)

Thank you very much.
Best regards,
Haiying

**Comment by GiudGiud** [comment, marked as answer]
Hello

Which input file are you talking about? kappa could be very different things depending on the physics. 
You can see its value by changing the `prop_values` field to the value you want. If you want something more complex than a constant, please let us know and we'll discuss what's best for your case

Guillaume

**Reply by abc-hy** [reply]
I am talking about solving the CH equation. What's the meaning of kappa when dealing with CH equation?

**Reply by GiudGiud** [reply]
@dschwen will know

**Reply by laagesen** [reply]
In this input file, `kappa` is the value of the gradient energy coefficient. You can get a good introduction to solving the Cahn-Hilliard equation using MOOSE by going through the following tutorial:

https://mooseframework.inl.gov/modules/phase_field/Tutorial.html
